kod/object/passive/itematt/weapatt/wapurge.kod

```diff
@@ -71,29 +71,28 @@ messages:
    %%% Effect Functions
       
    ModifyDamage(damage = 0, target = $, wielder = $, lData= $)
-   "Wielder has a 1% chance of blinding target with any hit."
+   "Wielder has a percent chance of purging target with any hit."
    {
       local oSpell;
 
       if random(1,100) < piEffect_percent
       {
-         oSpell = send(SYS,@FindSpellByNum,#num=SID_PURGE);
+         oSpell = Send(SYS,@FindSpellByNum,#num=SID_PURGE);
          
          %% Gotta have something to try to remove....
          if Send(target,@IsEnchanted)
          {
-            % Try to purge off random(25,75)% of the enchantments.
-            if send(oSpell,@DoPurge,#who=target,#iChance=random(25,75))
-            {
-               % Only tell our victim if they lost enchantments.
-               send(target,@MsgSendUser,#message_rsc=purger_worked_target,
-                    #parm1=send(wielder,@GetCapDef),#parm2=send(wielder,@GetName));
-            }
-
-            % Tell the wielder every time we fired so they think it's cool.
-            send(wielder,@MsgSendUser,#message_rsc=purger_worked_user,
-                 #parm1=send(target,@GetCapDef),#parm2=send(target,@GetName));
+            Send(target,@MsgSendUser,#message_rsc=purger_worked_target,
```

Comment:
I don't know about this.  The target is now getting an additional, duplicative message, as you noted:

Test1's weapon tears at the magic surrounding you!
A bright, holy light is trying to burn away your beneficial enchantments.

Reply:
I felt like this could make or break this PR. If we aren't okay with the standard purge message coming through, this would appear to become a more complicated matter. Though with my recent discovery of bItemCast, perhaps a bit of logic in `Purge` around the display of this message wouldn't be all that intrusive?

Reply:
Resolved via new logic in `purge.kod`.